src/crypto.c

```diff
@@ -126,7 +158,7 @@ int bip32_CKDpub(const serialized_extended_pubkey_t *parent,
     {  // make sure that heavy memory allocations are freed as soon as possible
         // compute point(I_L)
         uint8_t P[65];
-        if (0 > secp256k1_point(I_L, P)) return -1;
+        if (0 > secp256k1_point_unsafe(I_L, P)) return -1;
```

Comment:
Should be useful to explain (comments) why we can use `unsafe` function here

Reply:
Added comment in 3ba0aac4f29813e46cb2c531868d24130bd145b5, both here and in the `crypto_tr_tweak_pubkey` function.